app/Http/Controllers/CompetitionsController.php

```diff
@@ -137,4 +137,17 @@ public function message(Competition $competition, Contest $contest)
 
         return view('messages.show', compact('messages', 'competition'));
     }
+
+    /**
+     * Display the specified leaderboard.
+     *
+     * @param  \Gladiator\Models\Competition  $competition
+     * @return \Illuminate\Http\Response
+     */
+    public function leaderboard(Competition $competition)
+    {
+        $leaderboard = $this->manager->getLeaderboard($competition, $competition->users);
```

Comment:
I think you can call something like 
```php
$competition = $competition->users();
```
so then you can just pass `$competition` to the `getLeaderboard()` method instead of passing 2 vars, one of which is just an item of the other.